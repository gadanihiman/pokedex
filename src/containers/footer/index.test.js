import React from 'react';
import Footer from '.';
import renderer from 'react-test-renderer';

describe('Footer component', () => {
  test('Footer should successfully rendered', () => {
    const component = renderer.create(
      <Footer  />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
