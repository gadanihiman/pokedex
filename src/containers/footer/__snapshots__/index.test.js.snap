// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Footer should successfully rendered 1`] = `
<footer
  className="ant-layout-footer"
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  Pokemon Deck App created by
  <a
    href="https://www.linkedin.com/in/gadanihiman"
    rel="noopener noreferrer"
    target="_blank"
  >
     Gadani Himan Gurusinga
  </a>
   with
  <a
    href="https://pokeapi.co/"
    rel="noopener noreferrer"
    target="_blank"
  >
     PokeApi
  </a>
</footer>
`;
